pol-core/clib/esignal.cpp

```diff
@@ -15,7 +15,24 @@ namespace Pol
 {
 namespace Clib
 {
-std::atomic<bool> exit_signalled( false );
+std::atomic<bool> signal( false );
+std::atomic<int> code = 0;
+
+void signal_exit( int exit_code )
+{
+  code = exit_code, signal = true;
```

Comment:
Is saving 1 line of code worth having to think about what is going on with the comma operator here? (and in what order things happen)
```suggestion
  signalled_exit_code = exit_code;
  exit_signalled = true;
```

Reply:
funny how a typo can actually compile and still work fine... 🙃

Reply:
Addressed.